app/src/displays/related-values/related-values.vue

```diff
@@ -130,17 +133,28 @@ function getLinkForItem(item: any) {
 		z-index: 2;
 	}
 
-	&:not(.subdued):hover::before {
+	&:not(.disabled):hover::before {
 		opacity: 1;
 	}
 
-	&:not(.subdued):active::before {
+	&:not(.disabled):active::before {
 		background-color: var(--theme--background-accent);
 	}
 }
 
-.subdued {
+.render-template {
+	> .v-menu {
+		display: inline;
+
+		.toggle {
+			margin: $toggle-py $toggle-px;
+		}
+	}
+}
```

Comment:
Is this related to the same `toggle` element as above?
If so, let's merge & nest the styles for more clarity.